packages/vm/tests/api/runCall.spec.ts

```diff
@@ -224,6 +224,59 @@ tape('Ensure that Istanbul sstoreCleanRefundEIP2200 gas is applied correctly', a
   t.end()
 })
 
+tape('Ensure that Istanbul sstoreCleanRefundEIP2200 gas is applied correctly', async (t) => {
+  // setup the accounts for this test
+  const caller = new Address(Buffer.from('00000000000000000000000000000000000000ee', 'hex')) // caller addres
+  const address = new Address(Buffer.from('00000000000000000000000000000000000000ff', 'hex'))
+  const slot = Buffer.from('00'.repeat(32), 'hex')
+  const emptyBuffer = Buffer.from('')
+  // setup the vm
+  const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.London })
+  const vm = new VM({ common: common })
+  const code = '60008080F060005500'
+  /*
+      This simple code tries to create an empty contract and then stores the address of the contract in the zero slot.
+        CODE:
+          PUSH 0 
+          DUP1
+          DUP1
+          CREATE -> Stack is now [0,0,0] ([value, offset, length])
+          PUSH 0
+          SSTORE -> stack is now [0, <Created Address>], so this stores the <Created Address> at slot 0
+          STOP
```

Comment:
beautiful, really dig these opcode breakdowns

Reply:
Yes, can second that. 👍 🙂